Livecheckables/arduino-cli.rb

```diff
@@ -0,0 +1,3 @@
+class ArduinoCli
+  livecheck :regex => /^v?(\d+(?:\.\d+)+)b?$/
```

Comment:
```suggestion
  livecheck :regex => /^v?(\d+(?:\.\d+)+)$/
```

The trailing `b?` is unnecessary and, if anything, it could potentially lead to matching beta versions in the future, so it shouldn't be included. The standard regex for Git tags is fine here without modifications.